src/dune_rules/pkg_rules.ml

```diff
@@ -351,7 +351,22 @@ module Pkg = struct
         ]
   ;;
 
-  let deps_closure t = top_closure t.depends
+  let deps_closure t =
```

Comment:
Would it be worth a comment here to explain what this is for? Or would more familiar readers be likely to know just from the name?